docs: rewrite runs section (#1239)
apify/apify-docs: sources/platform/actors/development/builds_and_runs/runs.md

---

When you start an Actor, you create a run. A run is a single execution of your Actor with a specific input in a Docker container.

## Starting an Actor

You can start an Actor in several ways:

- Manually from the [Apify Console](https://console.apify.com/actors) UI
- Via the [Apify API](https://docs.apify.com/api/v2#/reference/actors/run-collection/run-actor)
- Using the [Scheduler](../../../schedules.md) provided by the Apify platform
- By one of the available [integrations](../../../integrations/index.mdx)

## Input and environment variables

The run receives input via the `INPUT` record of its default [key-value store](../../../storage/key_value_store.md). Environment variables are also passed to the run. For more information about environment variables check the [Environment variables](../programming_interface/environment_variables.md) section.

## Run duration and timeout

> Actor runs can be short or long-running. To prevent an indefinite runs, you can set a timeout. The timeout is specified in seconds and varies between Actor templates. If the run doesn't finish within the timeout, it's automatically stopped, and its status is set to `TIMED-OUT`.

Reviewer: I clarified it a bit:
```suggestion
Actor runs can be short or long-running. To prevent infinite runs, you can set a timeout. The timeout is specified in seconds, and the default timeout varies based on the template from which you create your Actor. If the run doesn't finish within the timeout, it's automatically stopped, and its status is set to `TIMED-OUT`.
```

Depending on the template used for your Actor, the default timeout values may differ.

> | Template name | Default timeout value in seconds |

Reviewer: I wonder if this table is really necessary, I don't think it's important to have the timeouts per template listed here. It will also be hard to maintain, as we'll have to update it every time we add, update or rename a template, and people will forget the templates are listed in this place.

Author: Yeah on one hand I think it would make sense to somehow give this information within docs, but on the other this table seems extremely unwieldy, I'll try to tinker with some other approaches

Author: maybe just something within prose like 

```
The default values for timeout can be:

* 3600 seconds
* 360 seconds

depending on the template you've used for your Actor
```

Reviewer: It doesn't necessarily have to be just these two values, it just so happened that all templates have either 360 or 3600 seconds. I would just skip mentioning the values altogether.
